packages/playground/website/src/components/site-manager-sidebar/index.tsx

```diff
@@ -0,0 +1,232 @@
+			return getLogoDataURL({
+				mime: 'image/png',
+				data: 'iVBORw0KGgoAAAANSUhEUgAAABQAAAAVCAYAAABG1c6oAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAALFSURBVHgBlVVLaxpRFL6OLpLdJIssko0kixQSqJAHNCilC6UbJf0H7T/IRhE3mo0Iumh/QV27SapLhVZ84EKpBXWnDC6kuPCB+ABRe76LZ5hYE9oPxpl7zz3f+c45915N4gUEAgHbcrnEY11PVS0WixYOh6vP+Zi2Tfp8vuDu7u4dEamHh4dif39fzvd6PdHpdMRsNtNoeB+NRuMvEvr9futisXg4Pj62uVwuQe9t8US5XBbpdFr0+33NbDa/i0Qi2l+Ea7KfTqdTpUf8C5LJpMjn8xp9fiC1sgyKgez7/5DV63WplGCl5yEUCqkYWPBDZEGqlXVvb48XicvLS6mA6iUwj/d0OpVlgA2EmFvDOplMvkKpGepWq1V8NBrJRSg6mnByciLa7bacw4Nv4PT0VBwcHEhyzAPUQEG1fHV9fZ1VSN3d2dmZ2NnZkcajoyPBaePtdrv1NLEGazkDY9MwpkxuUcPXFxcXcgJoNpsyOgNKOVir1XpiYx/YQT4cDt+C0IYUz8/PpREOlUpFd0I67AgYbQgGoMbIjEpnBaEKJ0SAAajVasIIQ/Gf2FitIaAKwsFmCkgNzWGgDAzYaEPLb14DpWvyAQg1drDb7bojd5CdjA3grQXydTPksSRUQZhlJ04doBOgq4MCHEUG25A+Z9VoNPD6pVAhH1ForgdvC4yhAGp4i3C3YUskEroAqAM57cXPSiwW+0ELHnHYuY7siDk0hFUbu41A3OVMJoN1cVwS8izP5/NPhUJBwyJExZVlTJfBW4uBjZ/L5bCVcOvcY86Mn1KpNLu5uclSHd7TUL26utIL7/F49O2EN4Kgy6w6lUoNiOwNX2FmjlYsFn87HI5v5HBL8tVutysJQGgEyPjEkIDqs/ehEV6v96PJZApi50PJ5o2tKMpgPB5/ofqHNn23EhqI5f8JqbBhTAE0elX5Mt2GP/1pl0TWF7M6AAAAAElFTkSuQmCC',
+			});
+		}
+		return `data:${logo.mime};base64,${logo.data}`;
+	};
+
+	const onLogoClick = () => {
+		onSiteChange();
+	};
+
+	return (
+		<aside className={classNames(css.siteManagerSidebar, className)}>
+			<header className={css.siteManagerSidebarHeader}>
+				{/* TODO move logo to components */}
+				<button
+					className={css.siteManagerSidebarLogoButton}
+					onClick={onLogoClick}
+				>
+					{/* TODO: move logo to component package */}
+					<svg
+						width="32"
+						height="32"
+						viewBox="0 0 32 32"
+						fill="none"
+						xmlns="http://www.w3.org/2000/svg"
+					>
+						<rect
+							width="10.4176"
+							height="10.4176"
+							rx="3.86258"
+							transform="matrix(0.829038 -0.559193 0.838671 0.544639 7.45703 24.1775)"
+							stroke="white"
+							strokeWidth="0.965644"
+						/>
+						<rect
+							width="13.2346"
+							height="13.2346"
+							rx="3.86258"
+							transform="matrix(0.829038 -0.559193 0.838671 0.544639 5.0918 18.9934)"
+							stroke="white"
+							strokeWidth="1.44847"
+						/>
+						<rect
+							width="17.451"
+							height="17.451"
+							rx="3.86258"
+							transform="matrix(0.829038 -0.559193 0.838671 0.544639 1.55371 11.6099)"
+							stroke="white"
+							strokeWidth="1.93129"
+						/>
+					</svg>
+				</button>
+			</header>
+			<div
+				className={classNames(
+					css.siteManagerSidebarSection,
+					css.siteManagerSidebarContent
+				)}
+			>
+				<h2 className={css.siteManagerSidebarSubtitle}>
+					WordPress Playground
+				</h2>
+				<label
+					className={classNames(
+						css.siteManagerSidebarLabel,
+						css.siteManagerSidebarListLabel
+					)}
+					htmlFor="site-list"
+				>
+					Your sites
+				</label>
+				<ul id="site-list" className={css.siteManagerSidebarList}>
+					{sites.map((site) => (
+						<li
```

Comment:
Let's use the same WordPress component as the site editor, even if that means literally copying it over:

![CleanShot 2024-08-02 at 15 18 10@2x](https://github.com/user-attachments/assets/516be01e-f6fd-4447-9750-7f0f4ebeef8c)

This would give us hover state, large click areas, focus state outlines, etc. for free.

Reply:
Actually, the component from the List View might be more useful here since we'll have an ellipsis menu eventually:

![CleanShot 2024-08-02 at 15 26 18@2x](https://github.com/user-attachments/assets/14d46581-8a7b-4c97-b35c-80de839bcb73)

Reply:
I switched to using WordPress components. It improved accessibility.